component/DateSelectionMonthlyCalendar.js

```diff
@@ -0,0 +1,102 @@
+import {useStoreon} from 'storeon/preact'
+import {html, createNextFreeSlotsForDateKey as __, createNextFreeSlotsForDateKey} from '../helper/index.js'
```

Comment:
Nem ez? Hogy működik ez most?
```suggestion
import {html, translator as __, createNextFreeSlotsForDateKey} from '../helper/index.js'
```